resources/views/PhpReboot/Magazine/articleDetails.blade.php

```diff
@@ -113,4 +152,25 @@
             </div>
         </div>
     </div>
-@endsection
\ No newline at end of file
+@endsection
+
+@section('pagescript')
```

Comment:
There should not be javascript code on template, it has to be on separate js file